tests/simulation/sim_test.go

```diff
@@ -0,0 +1,175 @@
+package simulation_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	simutils "github.com/zeta-chain/node/tests/simulation/sim"
+
+	"github.com/cosmos/cosmos-sdk/store"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/server"
+	cosmossimutils "github.com/cosmos/cosmos-sdk/testutil/sims"
+	cosmossim "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+	cosmossimcli "github.com/cosmos/cosmos-sdk/x/simulation/client/cli"
+)
+
+// AppChainID hardcoded chainID for simulation
+
+func init() {
+	simutils.GetSimulatorFlags()
+}
+
+const (
+	SimAppChainID  = "simulation_777-1"
+	SimBlockMaxGas = 815000000
+	SimDBBackend   = "goleveldb"
```

Comment:
simulation helper funtions only supports goleveldb and memdb backed for now
We can consider writing out own setup function to use pebbleDB if needed